website/docs/reference/widgets/table/README.md

```diff
@@ -207,6 +207,27 @@ As an API request with URL parameters:
 https://mock-api.appsmith.com/users?gender={{genderDropdown.selectedOptionValue}}
 ```
 
+## Freeze columns
+
+When your table has many columns, you may want to freeze important columns in place so that they're always visible, independent of scrolling.
```

Comment:
```suggestion
When your table has many columns, you may want to freeze and lock important columns in place so that they're always visible, independent of scrolling.
```
just adding the word lock for search